### DISCUSSION TITLE
Add outlook compatibility

### DISCUSSION THREAD

**Discussion by finaldzn** [GorvGoyl/ChatGPT-Writer#7] [Provide Feedback and Ideas]
In a work setting most companies uses Outlook as their email client.

A software like this but directly integrated into outlook could be very useful for some companies.

I even think some would be willing to pay for it.

I don't know tho, if using chatGPT will bring you good results in the long run compared to "rebuilding it " using the OpenAI API.

**Comment by GorvGoyl** [comment]
it's now supported on all sites https://chatgptwriter.ai/#how-to-use

enhanced support for outlook is coming soon.

**Reply by decodegu** [reply]
looking forward to it

**Comment by Vmadmax** [comment]
Any news here?

**Comment by jessedh** [comment]
I would be interested in licensing this as an outlook plugin for my team - is this on the roadmap? if so, when?

**Comment by gregoriopellegrino** [comment]
Up
